csharp/ql/lib/change-notes/2022-08-24-aps-net-core-controllers.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* ASP.NET Core controller definition made more precise. The amount of introduced taint sources or eliminated false positives should be low though, since the most common pattern is to derive all user defined ASP.NET Core controllers from the standard Controller class, which is not affected. 
```

Comment:
I'm not a native speaker, but doesn't `ASP.NET Core controller definition has been made more precise.` sound better?

Reply:
Neither am I. I have definitely forgot to add `was` at least. Fixed.